### ISSUE TITLE
Сomposer is broken and does not respond

### ISSUE THREAD

**Issue by jaitir** [wit-ai/wit#2520]
My composer is broken and does not respond to any commands, I can not connect the start module to test and deploy it.
When I try to move a block, the visualization of all the block disappears. But I can still change the contents of the blocks. 
The application is trained and responds normally to API commands

And if I create a new application, then the composer works in it.

Tell me, please, how can I solve this issue? ( App ID 551747280061902)

**Comment by jayyteee** [comment]
Hi @jaitir do you know if there are there any specific steps to take to reproduce this issue from a new app? It would help a lot if I could repro this issue to better debug it

**Comment by jaitir** [comment]
@jayyteee 
At first I assumed that this was due to the large number of nodes. But then I repeated the same architecture in the new application and everything works fine. This happened when I wanted to connect the start node at the beginning to deploy the project and after that I can’t do anything. I'm attaching a link to the screen recording, maybe it will help you.
[https://drive.google.com/file/d/1YuXBKzrfy-zhiCpar7yar2_humY9BqVV/view?usp=sharing](url)
